### PR TITLE
[AUTH-9284] Feature: gather locked accounts info

### PR DESCRIPTION
**Pull request by danielorihuela**
Hi guys, I am very excited since this is my very first contribution into this project.

I basically added a test to gather information about locked accounts following the guidelines of @chr0mag as mentiones in issue #474. Furthermore, I removed the system locked accounts as @chr0mag and @mboelen consensued in the same issue.

Any feedback is appreciated.

### CODE REVIEW

**Comment by danielorihuela** [comment]
Thanks for the explanation @Varbin. You are right. I fixed the implementation with your proposal. If you can take a look at it and give me more feedback it would be great! Thanks for your help.

File: include/tests_authentication [hunk 1]
```diff
@@ -859,23 +859,27 @@
...
                     FIND_P=$(passwd --all --status 2> /dev/null | ${AWKBINARY} '{ if ($2=="P" && $5=="99999") print $1 }')
                     FIND2=$(passwd --all --status 2> /dev/null | ${AWKBINARY} '{ if ($2=="NP") print $1 }')
+                    FIND3=$(passwd --all --status 2> /dev/null | ${AWKBINARY} '{ if ($2=="L") print $1 }' | sort | uniq)
                     ;;
             esac
         elif [ "${OS_REDHAT_OR_CLONE}" -eq 1 ]; then
             PREQS_MET="YES"
             FIND_P=$(for I in $(${AWKBINARY} -F: '{print $1}' "${ROOTDIR}etc/passwd") ; do passwd -S "$I" | ${AWKBINARY} '{ if ($2=="PS" && $5=="99999") print $1 }' ; done)
             FIND2=$(for I in $(${AWKBINARY} -F: '{print $1}' "${ROOTDIR}etc/passwd") ; do passwd -S "$I" | ${AWKBINARY} '{ if ($2=="NP") print $1 }' ; done)
+            FIND3=$(for I in $(${AWKBINARY} -F: '{print $1}' "${ROOTDIR}etc/passwd") ; do passwd -S "$I" | ${AWKBINARY} '{ if ($2=="L") print $1 }' | sort | uniq ; done)
```
Comment (Varbin) [inline include/tests_authentication:875]: My CentOS 8 passwd prints *LK* instead of *L*. Maybe both values should be checked here?

**Review by Varbin — CHANGES_REQUESTED** [review]

File: include/tests_authentication (hunk 1 above)
Comment (danielorihuela) [reply include/tests_authentication]: Perfect, added. It is starting to worry me how many of this differences might be between different distributions.

Comment (danielorihuela) [reply include/tests_authentication]: Thanks for you help, do you know if there exists more differences like that. If the answer is yes, maybe we should try and get the least amount of possibilities that cover the most of the linux distributions.

**Comment by mboelen** [comment]
Hi @danielorihuela - Congrats on your first pull request and thanks (also for your patience). I have reviewed it and it has been merged! A few minor changes have been applied, such as showing a suggestion instead of a warning.

With the change being implemented, it will be available in the next release.

**Pull request merged**